app/lib/constans.rb

```diff
@@ -0,0 +1,51 @@
+module Constants
```

Comment:
@benhalpern I like adding a `Constants` module for stuff like this, but if you want we can move the whitelist into `user.rb`.